### PR TITLE
Migrate from Carbon

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* Rename usages of `Test-SslCertificateBinding` to `Test-CSslCertificateBinding`.
+
+Update usages of `Get-SslCertificateBindings` to `Get-CSslCertificateBinding`. We removed the
+`Get-SslCertificateBindings` alias.
+
+Update usages of `Get-CSslCertificateBinding` that use the `IPAddress` or `Port` parameter with `-ErrorAction Ignore`.
+The `Get-CSslCertificateBinding` function now writes an error if searching for a binding and it doesn't exist.
+
+Update usages of `Remove-CSslCertificateBinding` with `-IPAddress '0.0.0.0'` and `-Port 443`. The function no longer
+deletes those bindings by default.
```
Comment (decoyjoe) [inline CHANGELOG.md]: This is confusing, maybe reword to something like:

> Update usages of Get-CSslCertificateBinding to include parameters -IpAddress '0.0.0.0' and -Port 443. The function no longer deletes those binding by default.

File: Carbon.Windows.HttpServer/en-US/about_Carbon.Windows.HttpServer.help.txt [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-    about_MODULE_NAME
+    about_Carbon.Windows.HttpServer
 
 SHORT DESCRIPTION
-    MODULE_NAME is a PowerShell module for...
+    Carbon.Windows.HttpServer is a PowerShell module for...
 
 LONG DESCRIPTION
-    The MODULE_NAME PowerShell module...
+    The Carbon.Windows.HttpServer PowerShell module...
```
Comment (decoyjoe) [inline Carbon.Windows.HttpServer/en-US/about_Carbon.Windows.HttpServer.help.txt]: Just making sure you meant to leave this file as-is.

File: README.md [hunk 3]
```diff
@@ -1,26 +1,48 @@
+<!-- markdownlint-disable -->
 # Overview
 
-The "MODULE_NAME" module...
+The Carbon.Windows.HttpServer module contains functions for configuring SSL/TLS/HTTPS certificate bindings on IP
+addresses and ports. Processes can't use an HTTPS certificate unless that certificate has been bound to a IP address and
```
Comment (decoyjoe) [inline README.md]: Would it be more accurate to say the processes uses the certificate from the certificate store? (There's nothing that stops a process from reading its certificate and private key from a file if it wanted to.)

> In order for a process to use an HTTPS certificate from the Windows certificate store, it must first be bound to an IP address and port.

File: TODO.md [hunk 4]
```diff
@@ -5,8 +5,8 @@ Pass the name of the module to the `ModuleName` parameter. The script will:
 
 * Rename this file to TODO.md (there are additional steps to take afterward)
 * Puts the default README.md file in place.
-* Rename every file that has `MODULE_NAME` in its name, replacing `MODULE_NAME` with the module name.
-* Replaces `MODULE_NAME` in every file with the module name.
+* Rename every file that has `Carbon.Windows.HttpServer` in its name, replacing `Carbon.Windows.HttpServer` with the module name.
```
Comment (decoyjoe) [inline TODO.md]: Remove this file?

**Review by decoyjoe — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (splatteredbits) [reply CHANGELOG.md]: Updated wording. Also changed to allow deleting all bindings, but requests confirmation from the user first.

File: Carbon.Windows.HttpServer/en-US/about_Carbon.Windows.HttpServer.help.txt (hunk 2 above)
Comment (splatteredbits) [reply Carbon.Windows.HttpServer/en-US/about_Carbon.Windows.HttpServer.help.txt]: I did not. Thank you.

File: README.md (hunk 3 above)
Comment (splatteredbits) [reply README.md]: Changed.

File: TODO.md (hunk 4 above)
Comment (splatteredbits) [reply TODO.md]: I cleared out all the completed contents. I'm finding a like a TODO.md file to note what to do when updating to the next minor and next major releases.
